bindata/network/multus/multus.yaml

```diff
@@ -521,50 +522,68 @@ spec:
             name: {{.CniSysctlAllowlist}}
             defaultMode: 0744
 {{if .RenderIpReconciler}}
+
 ---
-apiVersion: batch/v1
-kind: CronJob
+apiVersion: apps/v1
+kind: DaemonSet
 metadata:
-  name: ip-reconciler
+  name: whereabouts-reconciler
   namespace: openshift-multus
   labels:
     tier: node
```

Comment:
Question: is there a reason to have that label here?  It seems to be the only thing using that label:
~~~
[akaris@linux cluster-network-operator (whereabouts-reconciler-controller-yaml-fix)]$ oc get ds -A -l tier=node
NAMESPACE          NAME                     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR              AGE
openshift-multus   whereabouts-reconciler   6         6         6       6            6           kubernetes.io/arch=amd64   21m
[akaris@linux cluster-network-operator (whereabouts-reconciler-controller-yaml-fix)]$ oc get deployment -A -l tier=node
No resources found
[akaris@linux cluster-network-operator (whereabouts-reconciler-controller-yaml-fix)]$ oc get pods -A -l tier=node
NAMESPACE          NAME                           READY   STATUS    RESTARTS   AGE
openshift-multus   whereabouts-reconciler-2nghd   1/1     Running   0          21m
openshift-multus   whereabouts-reconciler-5smsm   1/1     Running   0          21m
openshift-multus   whereabouts-reconciler-cww4v   1/1     Running   0          21m
openshift-multus   whereabouts-reconciler-lfm8d   1/1     Running   0          21m
openshift-multus   whereabouts-reconciler-xd8nz   1/1     Running   0          21m
openshift-multus   whereabouts-reconciler-xnh67   1/1     Running   0          21m
~~~

Reply:
good catch! removing.